helm-charts/dnsbl-exporter/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
This here. I think this can go.

Reply:
We are creating the serviceAccount by a template with the ability the kubernetes admin could expose that account by custom labels/annotations just in case (very oftenly being used in AWS EKS, Redhat OKD.Openshift, etc). In vanilla Kubernetes it will look like the default one.
Helm's:
```yaml
kind: ServiceAccount
apiVersion: v1
metadata:
  name: dnsbl-dnsbl-exporter
  namespace: dnsbl
  uid: c6f21a7a-6cb9-4fff-9b20-fccacb24899b
  resourceVersion: '98953707'
  creationTimestamp: '2023-08-11T04:17:03Z'
  labels:
    app.kubernetes.io/component: dnsbl-exporter
    app.kubernetes.io/instance: dnsbl
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: dnsbl-exporter
    helm.sh/chart: dnsbl-exporter-0.0.1
  annotations:
    meta.helm.sh/release-name: dnsbl
    meta.helm.sh/release-namespace: dnsbl
```

default:
```yaml
kind: ServiceAccount
apiVersion: v1
metadata:
  name: default
  namespace: dnsbl
  uid: cfc1e125-1b0d-4f14-9730-28018e9b2e73
  resourceVersion: '98937766'
  creationTimestamp: '2023-08-11T03:39:48Z'
```